data_structures/linked-list/ll_merge.py

```diff
@@ -15,25 +15,3 @@ def merge_lists(list_one, list_two):
             list_two.insert_before(current_two.val, current.val)
             current, current_two = current._next, current_two._next
         return list_two.head.val
-
```

Comment:
The `if` and `else` blocks in this function are essentially identical, the only difference is the variables being used. To keep your code more DRY you might want to just put a condition about the variable assignments instead of the entire code block. You could also add another variable to keep track of which list to insert into.

(Commenting here cause cannot comment on relevant line)